mw-webapp/src/component/tag/Tag.tsx

```diff
@@ -1,6 +1,22 @@
 import clsx from "clsx";
 import styles from "src/component/tag/Tag.module.scss";
 
+/**
+ * Tag type enum
+ */
+export enum TagType {
+
+  /**
+   * Tag that used on way card
+   */
+  WAY_CARD_TAG = "wayCardTag",
+
+  /**
+   * Tag that used on User page and Way page
+   */
+  BASIC_TAG = "basicTag",
```

Comment:
Maybe PRIMARY_TAG? Don't know)

@tritonJS826 What do you think?

Reply:
For now I changed it to `PRIMARY_TAG`.